spec/acceptance/discovery_spec.rb

```diff
@@ -1,13 +1,58 @@
 require 'spec_helper_acceptance'
 
 describe 'Scenario: install foreman-proxy with discovery plugin'  do
+  root = case host_inventory['facter']['os']['name']
+         when 'Debian'
+           '/srv/tftp'
+         when 'Ubuntu'
+           if host_inventory['facter']['os']['release']['major'].to_f >= 20.04
+             '/srv/tftp'
+           else
+             '/var/lib/tftpboot'
+           end
```

Comment:
We've dropped Ubuntu 18.04 support so this can be simplified. If you copied it from somewhere, please point that out so we can fix it too.